src/HypothesisTests.jl

```diff
@@ -27,7 +27,7 @@ VERSION >= v"0.4.0-dev+6521" && __precompile__()
 module HypothesisTests
 
 using Distributions, Roots, StatsBase, Compat, Combinatorics
-using Rmath: pwilcox
+using Rmath: pwilcox, psignrank
```

Comment:
so these were right with the rather old rmath bindings from that package? statsfuns didn't expose something more recent?